### ISSUE TITLE
Module not found: Error: Cannot resolve module 'jquery'

### ISSUE THREAD

**Issue by jeerbl** [HubSpot/odometer#127]
Hello,

When `define` is defined and `define.amd` exists, it defines the module with a dependency on jQuery as you can see in the code below:

``` javascript
if (typeof define === 'function' && define.amd) {
  define(['jquery'], function() {
    return Odometer;
  });
} else if (typeof exports !== "undefined" && exports !== null) {
  module.exports = Odometer;
} else {
  window.Odometer = Odometer;
}
```

My project does not use jQuery and, when packaging with Webpack, I get the error

```
ERROR in ./~/odometer/odometer.js
Module not found: Error: Cannot resolve module 'jquery' in /Users/jbrunel/Documents/Projects/TrackInsight/view/src/public/map/node_modules/odometer
 @ ./~/odometer/odometer.js 644:4-646:6
```

Is there any reason why there is a dependency on jQuery when using `define` but not in the other cases?

I can provide a pull request to fix this if you are OK.

Thanks,
Jerome

**Comment by flex69** [comment]
Sounds good let's try probably you the solution do it .

**Comment by jeerbl** [comment]
Created PR #128.

**Comment by flex69** [comment]
It's ok fix it we are a team .

**Comment by jeerbl** [comment]
Fix merged in PR #128
